gpkit/tests/t_sub.py

```diff
@@ -150,6 +150,19 @@ def test_united_sub_sweep(self):
         firstcost = m.solve(verbosity=0)["cost"][0]
         self.assertAlmostEqual(firstcost, 1760, 3)
 
+    def test_skipfailures(self):
+        x = Variable("x")
+        x_min = Variable("x_{min}", [1, 2])
```

Comment:
oh cool, I either forgot or was never aware of this shorthand for `("sweep, [1, 2]")`. And I see it's already documented -- nice.